app/src/main/java/ar/com/wolox/android/cookbook/navigation/NavigationExtension.kt

```diff
@@ -0,0 +1,24 @@
+package ar.com.wolox.android.cookbook.navigation
+
+import androidx.navigation.NavOptions
+import ar.com.wolox.android.cookbook.R
+
+class NavigationExtension {
```

Comment:
We don't need a `class` for these methods. Just put them directly in the file and delete the `class`.

Reply:
What happened with this one. 🙄